src/detect-transform-md5.c

```diff
@@ -82,6 +71,11 @@ static int DetectTransformToMd5Setup (DetectEngineCtx *de_ctx, Signature *s, con
 static int DetectTransformToMd5Setup (DetectEngineCtx *de_ctx, Signature *s, const char *nullstr)
 {
     SCEnter();
+    if (g_disable_hashing) {
+        SCLogError(SC_ERR_HASHING_DISABLED, "MD5 hashing has been disabled, "
```

Comment:
Wonder if we need to be a bit more relax here. This would fail loading the set, but if we want rule writers to be comfortable including support like this into their set we may only want to fail if we're in strict mode?

Reply:
Is this an issue? I would expect people who do disable hashing to be in the small minority, and probably running more custom setups, so tweaking their input rules is probably a non-issue.